src/integration-test/java/openapi/OpenApiSpecificationIT.java

```diff
@@ -0,0 +1,85 @@
+package openapi;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.web.util.UriComponentsBuilder.fromPath;
+import static se.sundsvall.dept44.util.ResourceUtils.asString;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.ActiveProfiles;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+
+import se.sundsvall.businessengagements.BusinessEngagements;
+
+@SpringBootTest(
+	webEnvironment = RANDOM_PORT,
+	classes = BusinessEngagements.class,
+	properties = {
+		"spring.main.banner-mode=off",
+		"logging.level.se.sundsvall.dept44.payload=OFF"
+	}
+)
+@ActiveProfiles("it")
+class OpenApiSpecificationIT {
+
+	private static final YAMLMapper YAML_MAPPER = new YAMLMapper();
+
+	@Value("${openapi.name}")
+	private String openApiName;
+
+	@Value("${openapi.version}")
+	private String openApiVersion;
+
+	@Value("classpath:/openapi.yml")
+	private Resource openApiResource;
+
+	@Autowired
+	private TestRestTemplate restTemplate;
```

Comment:
Why TestRestTemplate? Why not WebTestClient?

(I read somewhere that the Spring team recommends WebClient/WebTestClient instead of RestTemplate/TestRestTemplate)

Reply:
Hm, this class is just copied from another service that verified the API in a better way than the replaced class. A quick search shows that TestRestTemplate is used in at least 23 classes that tests the OpenAPI specification. If we are to change, there should be a task for changing to WebTestClient for all services that uses TestRestTemplate today. Is it worth it, I don't know?

But I can change to WebTestClient in this service.

Reply:
You touched it last, you'll fix!

Reply:
All I'm saying is.. There exists a tool to quickly make tons of pull requests.. ;)